### PR TITLE
DMP-738 - implement GET /cases/{caseId}/transcripts

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
https://tools.hmcts.net/jira/browse/DMP-738

### CODE REVIEW

File: src/main/resources/openapi/cases.yaml [hunk 1]
```diff
@@ -183,6 +183,34 @@ paths:
...
+          description: "case_id is the internal cas_id of the case."
+          required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/transcripts'
+        '500':
```
Comment (davet1985) [inline src/main/resources/openapi/cases.yaml:207]: Can you add in the 404 response for a non-existent case id

**Review by davet1985 — CHANGES_REQUESTED** [review]
Couple of minor comments

File: src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+        UserAccountEntity testUser = dartsDatabase.getUserAccountStub()
+            .createAuthorisedIntegrationTestUser(courthouseEntity);
+        when(mockUserIdentity.getEmailAddress()).thenReturn(testUser.getEmailAddress());
+
+
+    }
+
+    @Test
+    void casesSearchGetEndpoint() throws Exception {
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java]: name updated

File: src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        String expected = TestUtils.removeTags(TAGS_TO_IGNORE, getContentsFromFile(
+            "tests/cases/CaseControllerGetCaseTranscriptsTest/casesSearchGetEndpointOneObjectReturned.json"));
+        MvcResult mvcResult = mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isOk())
+            .andReturn();
+        String actualResponse = TestUtils.removeTags(TAGS_TO_IGNORE, mvcResult.getResponse().getContentAsString());
+        JSONAssert.assertEquals(expected, actualResponse, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    void casesSearchGetEndpointTwoObjectsReturned() throws Exception {
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java]: name updated

File: src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    void casesSearchGetEndpoint() throws Exception {
+
+        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, "25");
+
+        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isNotFound());
+
+    }
+
+    @Test
+    void casesSearchGetEndpointOneObjectReturned() throws Exception {
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseTranscriptsTest.java]: name updated

File: src/main/resources/openapi/cases.yaml (hunk 1 above)
Comment (cmagowan-hmcts) [reply src/main/resources/openapi/cases.yaml:217]: definition added

**Review by davet1985 — APPROVED** [review]

**Pull request merged**